sdg/components.py

```diff
@@ -182,17 +182,28 @@ def get_git_host(repo_url):
         print("No credentials provided for taxonomy repo, assuming public repo...")
 
     ca_cert_path = os.getenv("SDG_CA_CERT_PATH")
+    ssl_cert_dir = os.getenv("SSL_CERT_DIR")
+    ssl_cert_file = os.getenv("SSL_CERT_FILE")
```

Comment:
can we get a comment here that distinguishes between the taxonomy vs platform approach?